src/main/java/core/basesyntax/maketransaction/operations/Buy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.maketransaction.operations;
+
+import core.basesyntax.identities.Fruit;
+import core.basesyntax.identities.Storage;
+import core.basesyntax.maketransaction.IFruitOperation;
+import core.basesyntax.maketransaction.Transaction;
+import java.time.LocalDate;
+
+public class Buy implements IFruitOperation {
+    @Override
+    public boolean apply(Transaction transaction, Storage storage) {
```

Comment:
Please check Thursday's video. I am not passing the storage as a method parameter. In such a way, you are moving from implementation to interface. It would be best if you moved from the interface to implementation